CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,78 @@
+# Scipian Contributor Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers of Scipian pledge to making participation in our 
+project and our community a harassment-free experience for everyone, regardless 
+of age, body size, disability, ethnicity, sex characteristics, gender identity 
+and expression, level of experience, education, socio-economic status, 
+nationality, personal appearance, race, religion, or sexual identity and 
+orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Our Responsibilities
+
+Scipian project maintainers are responsible for clarifying the standards of 
+acceptable behavior and are expected to take appropriate and fair corrective 
+action in response to any instances of unacceptable behavior.
+
+Scipian project maintainers have the right and responsibility to remove, edit, 
+or reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing Scipian or its community. Examples of
+representing a project or community include using an official Scipian e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of Scipian may be
+further defined and clarified by Scpian project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the Scipian project team at TODO(NAL):[INSERT EMAIL ADDRESS]. 
```

Comment:
Do we need to create a ServiceNow ticket to get this email alias created?

Reply:
Not sure, but I think to begin, we should just my email address for now.

Reply:
Agreed. I think we should have an email address added before committing this. We should also create an issue to follow up on creating a Scipian email alias.

Reply:
Yes, but it should be on the `scipian.io` domain or maybe a hosted mailing list. No need to get into details here, let's create another issue to look into public mailing lists and etc.

Reply:
@dragan so for now, you want me to put your SAP email here?

Reply:
Yes